lib/ansible/modules/crypto/entrust/ecs_certificate.py

```diff
@@ -0,0 +1,906 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c), Entrust Datacard Corporation, 2019
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: ecs_certificate
+author:
+    - Chris Trufan (@ctrufan)
+version_added: '2.9'
+short_description: Request SSL/TLS certificates with the Entrust Certificate Services (ECS) API
+description:
+    - Create, reissue, and renew Certificates with the Entrust Certificate Services (ECS) API.
+    - Requires credentials for the L(Entrust Certificate Services, https://www.entrustdatacard.com/products/categories/ssl-certificates) (ECS) API.
+    - In order to request a certificate, the domain and organization used in the certificate signing request must be already
+      validated in the ECS system. It is I(not) the responsibility of this module to perform those steps.
+notes:
+    - C(path) must be specified as the output location of the certificate.
+requirements:
+    - cryptography >= 1.6
+options:
+    backup:
+        description:
+            - pathination to store a backup of the initial certificate, if I(path) pointed to an existing file certificate.
+        type: bool
+        default: False
```

Comment:
Just a quick comment, booleans used in the documentation YAML itself should also be true/false/yes/no:
```suggestion
        default: false
```

Reply:
Ah, my apologies, didn't realize your earlier comment was about the case of the word. Thanks!